src/multi-level-checkbox-editor/multi-level-checkbox-editor.less

```diff
@@ -133,3 +136,20 @@
     }
 }
 
+.empty-nodebox {
+    display: flex;
+    margin: 10px 12px;
+    width: 180px;
+    border: 1px solid #D7D7D7;
```

Comment:
I think we have variables for these colors

Reply:
changed